llvmlite/tests/test_binding.py

```diff
@@ -2511,7 +2511,7 @@ def test_populate(self):
         pm.add_instruction_combining_pass()
         pm.add_licm_pass()
         pm.add_sccp_pass()
-        pm.add_sroa_pass()
+        # pm.add_sroa_pass()
```

Comment:
This still seems to work?
```suggestion
        pm.add_sroa_pass()
```